### DISCUSSION TITLE
How to use vue-i18n 9 in both legacy mode and non-legacy mode?

### DISCUSSION THREAD

**Discussion by Kasheftin** [intlify/vue-i18n#1252] [Q&A]
We have migrated to vue3, and the most components still use options api. Hovewer the new ones use composition api. And we use inline translations meaning every component might have it's own <i18n lang="yaml"> block. How to set up vue-i18n for our case?

**Comment by Theiaz** [comment]
I'm wondering the same. Vues composition api has also been made to mix options and composition api, during slow migration to composition.

This tool seems like it does not support it?

**Reply by Theiaz** [reply]
Okay stumbled upon the following post: https://github.com/intlify/vue-i18n-next/discussions/605#discussioncomment-1122806 which clarifies the case.
